uportal-war/src/main/webapp/WEB-INF/flows/permissions-administration/deletePermission.jsp

```diff
@@ -0,0 +1,121 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+
+<c:set var="n"><portlet:namespace/></c:set>
+
+<portlet:actionURL var="removeUrl">
+    <portlet:param name="execution" value="${flowExecutionKey}" />
+    <portlet:param name="_eventId" value="remove"/>
+</portlet:actionURL>
+
+<portlet:actionURL var="cancelUrl">
+  <portlet:param name="execution" value="${flowExecutionKey}" />
+  <portlet:param name="_eventId" value="cancel"/>
+</portlet:actionURL>
+
+<!--
+PORTLET DEVELOPMENT STANDARDS AND GUIDELINES
+| For the standards and guidelines that govern
+| the user interface of this portlet
+| including HTML, CSS, JavaScript, accessibilty,
+| naming conventions, 3rd Party libraries
+| (like jQuery and the Fluid Skinning System)
+| and more, refer to:
+| http://www.ja-sig.org/wiki/x/cQ
+-->
+    
+<!-- Portlet -->
+<div class="fl-widget portlet ptl-mgr view-confirmremove" role="section">
+
+  <!-- Portlet Title -->
+  <div class="fl-widget-titlebar titlebar portlet-titlebar" role="sectionhead">
+  
+    <div class="breadcrumb">
+        <c:forEach items="${ breadcrumbs }" var="breadcrumb" varStatus="status">
+            <portlet:actionURL var="breadcrumbUrl">
+                <portlet:param name="execution" value="${flowExecutionKey}" />
+                <portlet:param name="_eventId" value="${ breadcrumb.key }"/>
+            </portlet:actionURL>
+            <span class="breadcrumb-${ status.index + 1 }">
+                <a href="${ breadcrumbUrl }">${ fn:escapeXml(breadcrumb.value )}</a>
+            </span>
+            <span class="separator">&gt; </span>
+        </c:forEach>
+    </div>
+  
+    <h2 class="title" role="heading">
+      <spring:message code="remove.permission"/>
+    </h2>
+  </div> <!-- end: portlet-titlebar -->
+  
+  <!-- Portlet Content -->
+  <div class="fl-widget-content content portlet-content" role="main">
+
+    <!-- Portlet Section -->
+    <div class="portlet-section" role="region">
+      <div class="titlebar">
+         <h3class="title" role="heading"><spring:message code="remove.permission.confirmation" arguments="${permissionType},${ activity.name },  ${target.name }, ${principalName}"/></h3>
+        
+      </div>
+    </div> <!-- end: portlet-section -->
+   
+    <div class="buttons">
+        <a href="${removeUrl}" class="button btn btn-default" name="_eventId_remove" id="${n}deletePermissionButton"><spring:message code="remove"/></a>
+      <a href="${cancelUrl}" class="button btn btn-default"><spring:message code="cancel"/></a>
+    </div>
+    
+
+    </div> <!-- end: portlet-content -->
+        
+</div> <!-- end: portlet -->
+
+<script type="text/javascript">
+up.jQuery(function() {
+    var $ = up.jQuery;
+    
+    $(document).ready(function(){
+        $('#${n}deletePermissionButton').click(function(e){
+            var principals = [<c:forEach items="${ principals }" var="principal" varStatus="status">'${ principal }'${ status.last ? '' : ', '}</c:forEach>];
```

Comment:
Hey @jameswennmacher , you're correct.  Principals was used at one point but forgot to take it out.  Good catch!  Will respond when the pull request is updated.